core/domain/suggestion_services.py

```diff
@@ -344,16 +344,17 @@ def get_translation_contribution_stats_models(
     Raises:
         Exception. The stats models do not exist for the given IDs.
     """
-
     stats_models = (
-    suggestion_models.TranslationContributionStatsModel.get_multi(
-        list(stats_ids)))
+        suggestion_models.TranslationContributionStatsModel.get_multi(
+            list(stats_ids)))
 
-    if strict:
-        for model in stats_models:
-            if model is None:
-                raise Exception(
-                    'The stats models do not exist for the given IDs.')
+    if not strict:
+        return stats_models
+
+    for model in stats_models:
+        if model is None:
+            raise Exception(
+                'The stats models do not exist for the given IDs.')
```

Comment:
Better to say exactly which ID, for debugging purposes.

Reply:
Something like:

```python
    for index, model in enumerate(stats_models):
        if model is None:
            raise Exception(
                'The stats models do not exist for the stats_id %s.' % stats_ids[index]
            )
```

because currently, we are following this pattern but if you have a better idea feel free to use that!

Reply:
Done